test/dates/io.jl

```diff
@@ -432,3 +432,14 @@ end
 
 # Issue #21504
 @test isnull(tryparse(Dates.Date, "0-1000"))
+
+# Issue #22100
```

Comment:
Perhaps make a testset of this issue tests? Simplifies for whoever converts this file to `@testset`s later on